sdk/src/transaction/sanitized.rs

```diff
@@ -58,9 +60,15 @@ impl SanitizedTransaction {
             return Err(TransactionError::AccountLoadedTwice);
         }
 
+        let is_simple_vote_tx = is_simple_vote_tx.unwrap_or_else(|| {
+            let mut ix_iter = message.program_instructions_iter();
+            ix_iter.next().map(|(program_id, _ix)| program_id) == Some(&crate::vote::program::id())
```

Comment:
Don't we need to check that there are no other instructions following the vote?

Reply:
Well, I did have that initially but noticed that we don't enforce that here: https://github.com/solana-labs/solana/blob/c0952831be0e4916a477613d76e2a28de5df5d8c/programs/vote/src/vote_transaction.rs#L46

I would be happy to add that check, but didn't feel that we needed it. What do you think?